frontend/src/components/nodes/NodeRow.tsx

```diff
@@ -16,31 +16,45 @@ export default class extends React.PureComponent<Props> {
       <Row className="node-row mx-0">
         <Col md="auto" xs="1" className="pr-0">
           <img
-            src="/static/images/icon-m-block.svg"
+            src={
+              node.isValidator
+                ? "/static/images/icon-m-node-validator-status.svg"
+                : "/static/images/icon-m-node-non-validator-status.svg"
+            }
             style={{ width: "15px" }}
           />
         </Col>
-        <Col md="7" xs="6">
+        <Col md="7" xs="7">
           <Row>
-            <Col className="node-row-title">{node.moniker}</Col>
+            <Col className="node-row-title">
+              @{node.accountId}{" "}
+              <span className="node-status"> {node.status}</span>
+            </Col>
           </Row>
           <Row>
             <Col className="node-row-text">
               <Row>
-                <Col md="auto">#{node.lastHeight}</Col>
-                <Col md="auto" className="pl-0">
+                <Col md={5}>
+                  <img
+                    src="/static/images/icon-m-size.svg"
+                    style={{ width: "12px" }}
+                  />
+                  {`@  ${node.agentName} | ver.  ${node.agentVersion} build  ${
```

Comment:
1. Let's remove `@  `. It is not an account, so we don't want to confuse people.
2. Remove the extra space after `ver.`